openedx/features/edly/utils.py

```diff
@@ -1,5 +1,29 @@
 import jwt
 from django.conf import settings
+from django.contrib.sites.shortcuts import get_current_site
+from django.db.models import Q
+
+from openedx.features.edly.models import EdlySubOrganization
+
+
+def user_has_edly_organization_access(request):
+    """
+    Returns True if the edly sub organization of the requested URL belongs to the user's linked edly sub organizations.
+
+    Arguments:
+        request: HTTP request object
+
+    Returns:
+        bool: Returns True if User has Edly Organization Access Otherwise False.
+    """
+    if not request.user.edly_profile:
+        return False
+
+    current_site = get_current_site(request)
+    if EdlySubOrganization.objects.filter(Q(lms_site=current_site) | Q(studio_site=current_site)).count() == 1:
```

Comment:
We need to check if the `edly_sub_organization` linked to current request site is linked with requesting user's `edly_profile` or not linked.
```python
edly_sub_organization = EdlySubOrganization.objects.filter(Q(lms_site=current_site) | Q(studio_site=current_site))
if edly_sub_organization and edly_sub_organization in equest.user.edly_profile.edly_sub_organizations_set.all():
    return True

return False
```

Reply:
`AttributeError: 'EdlyUserProfile' object has no attribute 'edly_sub_organizations_set` instead i checked using the `get_linked_edly_sub_organizations` method.

Reply:
👍